share/poly.go

```diff
@@ -40,7 +41,11 @@ func (p *PriShare) Hash(s kyber.HashFactory) []byte {
 }
 
 func (p *PriShare) String() string {
-	return fmt.Sprintf("{%d:%p}", p.I, p.V)
+	data, err := p.V.MarshalBinary()
+	if err != nil {
+		panic("couldn't marshal point")
+	}
+	return fmt.Sprintf("{%d:%s}", p.I, hex.EncodeToString(data))
```

Comment:
Sorry - should've thought of this 1 week ago...

```suggestion
	return fmt.Sprintf("{%d:%s}", p.I, p.V)
```

Reply:
Fine for me 👍

Reply:
Marked as resolved, but the commit wasn't included..? I'm fine in using the simple `%s` solution and think this should be the chosen approach.

Reply:
Using the simple `%s` is fine with me too.

Reply:
Ok. So please update the PR to just use `%s` and let's merge it.

Reply:
Thx a lot ! I'll close this PR and repropose a new one.

Reply:
I have proposed a new PR #459, plz help to merge it. Many thanks.